onnxruntime/core/providers/openvino/backend_utils.cc

```diff
@@ -1,21 +1,107 @@
 // Copyright (C) Intel Corporation
 // Licensed under the MIT License
-
 #include <algorithm>
 #include <sstream>
 #include <fstream>
 #include <utility>
 
+#include <filesystem>
```

Comment:
@saurabhkale17 please run cpplint to ensure this issue does not occur with MSFT CI Pipelines